sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQuerySuite.scala

```diff
@@ -1106,6 +1107,54 @@ class StreamingQuerySuite extends StreamTest with BeforeAndAfter with Logging wi
     }
   }
 
+  test("union in streaming query of append mode without watermark") {
```

Comment:
nit: missing jira number. Maybe use `SQL union` to make it clear?

Reply:
Thanks, done in 658f30a.